gitlab/v4/objects/iterations.py

```diff
@@ -16,11 +16,11 @@ class GroupIterationManager(ListMixin, RESTManager):
     _path = "/groups/{group_id}/iterations"
     _obj_cls = GroupIteration
     _from_parent_attrs = {"group_id": "id"}
-    _list_filters = ("state", "search", "include_ancestors")
+    _list_filters = ("state", "search", "in", "include_ancestors")
```

Comment:
LGTM @cristianocasella, but while we're doing this, maybe it makes sense to add the remaining missing filters? See https://docs.gitlab.com/ee/api/iterations.html.

Reply:
that's a great idea, let me update this